source/core_mqtt_serializer.c

```diff
@@ -2327,7 +2334,8 @@ MQTTStatus_t MQTT_DeserializeAck( const MQTTPacketInfo_t * pIncomingPacket,
 
             /* Any other packet type is invalid. */
             default:
-                LogError( ( "IotMqtt_DeserializeResponse() called with unknown packet type:(%02x).", pIncomingPacket->type ) );
+                LogError( ( "IotMqtt_DeserializeResponse() called with unknown packet type:(%02x).",
+                            pIncomingPacket->type ) );
```

Comment:
Shouldn't there be one here too?
```suggestion
                            ( unsigned int ) pIncomingPacket->type ) );
```

Reply:
Good point. I think it should be signed though.